civihr_default_theme/templates/user/page--user--edit--onboarding.tpl.php

```diff
@@ -1,15 +1,15 @@
 <?php
 
 /**
- * @var array $page Contains all variables and details for the page
- * @var string $container_class
- */
-
-/**
+ * @var array
+ *   $page Contains all variables and details for the page
+ * @var string
+ *   $container_class
```

Comment:
Do we need this parameters?, I don't see them being used in the function.

Reply:
You're right, $container_class isn't used. $page is however, so I'll leave that